<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="handleloginPage1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="1">
           <widget class="QPushButton" name="LogSellerPushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Войти как сотрудник</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="LoginLabel">
            <property name="text">
             <string>Введите логин:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="PasswordLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Введите пароль:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="LogAdminPushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Войти как администратор</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="LoginLineEdit"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="Operation">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="PasswordLineEdit">
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="AdminChoice">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="EmployeeManagement">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Управление сотрудниками</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="WarehouseManagement">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Управление складом</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="BackButton">
          <property name="text">
           <string>Выход</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="WarehousePage4">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="3" column="0">
           <widget class="QPushButton" name="ViewWareHousePushButton">
            <property name="text">
             <string>Поиск лекарства</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="RemoveMedizinePushButton">
            <property name="text">
             <string>Удалить лекарство</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QScrollBar" name="verticalScrollBar_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="AddMedicinePushButton">
            <property name="text">
             <string>Добавить лекарство</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget_2"/>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="Cancel_Warehouse">
            <property name="text">
             <string>Назад</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ManagementPage3">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0">
           <widget class="QPushButton" name="SearcheEmployeePushButton">
            <property name="text">
             <string>Поиск сотрудника</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget"/>
          </item>
          <item row="0" column="1">
           <widget class="QScrollBar" name="verticalScrollBar">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="AddEmployeePushButton">
            <property name="text">
             <string>Добавить сотрудника</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="RemoveEmployeePushButton">
            <property name="text">
             <string>Удалить сотрудника</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="Cancel_Employee">
            <property name="text">
             <string>Назад</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LoginLineEdit</tabstop>
  <tabstop>PasswordLineEdit</tabstop>
  <tabstop>LogAdminPushButton</tabstop>
  <tabstop>LogSellerPushButton</tabstop>
  <tabstop>AddEmployeePushButton</tabstop>
  <tabstop>RemoveEmployeePushButton</tabstop>
  <tabstop>SearcheEmployeePushButton</tabstop>
  <tabstop>Cancel_Employee</tabstop>
  <tabstop>EmployeeManagement</tabstop>
  <tabstop>WarehouseManagement</tabstop>
  <tabstop>BackButton</tabstop>
  <tabstop>AddMedicinePushButton</tabstop>
  <tabstop>RemoveMedizinePushButton</tabstop>
  <tabstop>ViewWareHousePushButton</tabstop>
  <tabstop>Cancel_Warehouse</tabstop>
  <tabstop>tableWidget_2</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
